[Feat] Mypage_Backend

kb4954 opened the pull request:
**Check List :memo:**
- [x] Pull Request 제목 : [유형] 변경 사항 요약
- [x] 설명 & 이슈 번호 작성

**설명 (필수)**

이 PR에서 어떤 변경을 했는지 상세히 설명해주세요.
관련된 Issue 번호를 포함하세요.

1. #5 

2. 변경 사항
이전 브랜치 내용 전부 옮김
리뷰리스트, 북마크, 내그룹리스트 html,css 추가 
js 작업중

3. 스크린샷 (선택)

<br>

---

**review 참고 사항 (선택)**

추가로 알릴 내용이 있다면 여기에 작성해주세요.

1.

2.

sharehyun committed:
login통합

resader1 committed:
0708am

전 페이지 base.html extends로 변경.
js추출.

회원가입 id중복확인 기능 추가
---
1.중복확인 미진행시 회원가입 진행 불가 및 경고창 팝업
2.중복확인 후 아이디 변경시 중복확인 재진행 안내문 팝업 후 중복확인 기록 삭제

sharehyun committed:
로그인기능 양식수정

resader1 committed:
회원정보페이지 재추가

member/update/user_id 입력시 진입 가능
취소 버튼 클릭 - confirm(수정 취소?) 수락시 뒤로가기

미로그인 상태에서 진입시 - 로그인 하라는 경고창 뜨고 login페이지로 넘어감

타 계정 정보창으로 진입시 - 본인 정보만 수정 가능하다는 경고창 뜨고 홈으로 넘어감(기존엔 로그인 페이지로 넘어갔으나 로그인 된 상태에서 로그인 창으로 넘어가지는 현상으로 인하여 수정)

null4848 committed:
feat: 별점 표시

hwk3005 committed:
250708 백엔드_html 수정본 업로드

250708 백엔드_html 수정본 업로드

sharehyun committed:
메인페이지 top5출력

sharehyun committed:
session_id -> user_id

로그인시 우측 변경되지 않던 문제 해결

resader1 committed:
회원가입 취소시 홈으로

join2.js 223줄 수정

sharehyun committed:
member 통합전 url오류 수정

추후 다시 변경 예정

resader1 committed:
이용약관 기능추가

join1 모두 동의 기능 추가

kb4954 committed:
feat:삭제 활성화 완료

북상세페이지와 연동해서 상세페이지에서도 삭제되는거 확인완료

null4848 committed:
feat: 답글 DB 생성

답글 DB에 생성되는데
아직 뜨지는않음 이거 수정필요

kb4954 committed:
feat: 북마크 db 마이페이지에 연결중

hwk3005 committed:
250708 백엔드_Form제출시 DB저장, 그룹목록에 추가한 그룹 표시

1. 그룹만들기 Form제출시
  1) DB에 저장 완료
  2) 그룹목록, 참여 중인 그룹에 표시 완료

2.  그룹만들기-완료 버튼 클릭 시 필수항목 미입력 시 alert, focus 적용

3. 그룹목록-해시태그 css 수정

null4848 committed:
feat: 댓글DB 연결완료

HTML에 뜨게함
근데 AJAX 로 받아오는게 좋을 것 같음
추후 수정 예정

kb4954 committed:
feat:북마크 삭제기능

리뷰페이지 부분 별점  html 수정했는데 반영안돼서 나중에 브랜치 합치고 진행 예정

null4848 committed:
feat: 좋아요 버튼 DB

좋아요 버튼 눌렀을때
값이 DB에 저장되게 함
새로고침하면 버튼 클릭 이벤트가 초기화되서 또 클릭할 수 있게 됨 ..
수정 필요

kb4954 committed:
feat:닉네임에 유저이름 연결

null4848 committed:
feat: 리뷰, 댓글 삭제, DB 삭제 반영

리뷰 댓글, 삭제 기능 구현
삭제 시 alert 창뜨게함
삭제한 거 DB에 반영해서 값 달라지게 함

resader1 committed:
0709pm

중복확인재진행 경고창 제거
성별 강제선택 제거

sharehyun committed:
검색바수정

sharehyun committed:
최신버전

kb4954 committed:
feat: 상세이동

kb4954 committed:
Merge branch '3-feat-cscenter-backend' into 5-feat-mypage01_backend

sharehyun committed:
Merge branch '5-feat-mypage_backend' into 3-feat-cscenter-backend

null4848 committed:
feat: 백엔드 구현

댓글, 리뷰 삭제, 북마크 수 DB 저장
book db review_count, bookmark_count, rating 저장후
총평점 출력
페이지네이션 적용

sharehyun committed:
Merge remote-tracking branch 'origin/5-feat-mypage01_backend' into 3-feat-cscenter-backend

null4848 committed:
Merge branch '3-feat-cscenter-backend' into 4-feat-book-search-backend

null4848 committed:
feat:  별점 평균 전송

sharehyun committed:
별점->각자 ranking 기준,마이페이지 추가

null4848 committed:
setting

null4848 committed:
feat: 별점 표시 변경

kb4954 committed:
feat: 유현님과 브랜치 병합

브랜치 병합 후 교환독서부분 연결
리뷰박스 클릭시 도서 상세페이지로 이동

sharehyun committed:
Merge branch '4-feat-book-search-backend' into 3-feat-cscenter-backend

sharehyun committed:
Remove pycache files from git tracking

sharehyun committed:
Remove cached .pyc files and ignore pycache

hwk3005 committed:
250709 백엔드_그룹만들기 백엔드 구현 완료(도서api연결, Form제출시 DB최종저장)

1. 그룹만들기
  1) 책 선택-모달창
    - 도서 api 연결
    - 책제목/저자 등 키워드 검색 시 검색목록 표시
    - 책 선택버튼 클릭 시 모달창 닫히고 책 선택됨 (변경도 가능)
  2) Form제출 시 shareMain-ReadingGroup db에 최종 저장

2. 메인페이지
  1) 참여중인 그룹
    - 만든 그룹 추가
    - 책 이미지 안 띄워지던 오류 해결
  2) 그룹 목록
    - 만든 그룹 최신순으로 정렬
    - 해시태그, 책 이미지 안 띄워지던 오류 해결

hwk3005 committed:
250709 교환독서 title 적용

250709 교환독서 title 적용

sharehyun committed:
reviewlist->review 링크수정

null4848 committed:
feat: 메세지 이중 출력 수정, 사진 파일 정리

kb4954 committed:
feat: shareReading 삭제함

혜원님 브랜치 병합해서 진행할 예정

kb4954 committed:
Merge remote-tracking branch 'origin/7-feat-share_main-addgroup-backend' into 5-feat-mypage01_backend

hwk3005 committed:
250709 백엔드_Main 검색창 기능 구현, 로그인 관련 기능 적용 중

1. 그룹만들기
  1) 책 선택-모달창
    - 선택 버튼 클릭 시 모달창 닫히고 책 추가 css, tooltip 수정
  2) 로그인 했을 때만 그룹만들기 가능하도록 적용 중

2. 메인페이지
  1) 참여중인 그룹
    - 로그인 했을 때만 표시되도록 기능 적용 중 (member)
  2) 그룹 목록
    - 검색창: 검색 시 검색내용 포함한 그룹 검색 (id 최신순 정렬)
    - 검색창: css 수정 (공지사항p 통일)

kb4954 committed:
feat:북마크페이지 도서상세페이지와 연결

카테고리탭에 북마크 카운트도 가져옴

hwk3005 committed:
Fix merge conflict in Share_AddGroup.html

hwk3005 committed:
member 머지 충돌 해결

hwk3005 committed:
Merge remote-tracking branch 'origin/8-feat-member' into 7-feat-share_main-addgroup-backend

hwk3005 committed:
250709 백엔드_member브랜치 머지 완료, 로그인 관련 기능 적용 중

0. member into shareMain 브랜치 머지 완료

2. 메인페이지
  1) 참여중인 그룹
    - 로그인 했을 때만 표시되도록 기능 적용 완료

kb4954 committed:
feat: 페이지네이터 활성화

kb4954 committed:
Merge remote-tracking branch 'origin/7-feat-share_main-addgroup-backend' into 5-feat-mypage01_backend

sharehyun committed:
북디테일 상세정보밑 교환독서그룹

sharehyun committed:
짜잘짜잘...

제목 말줄임표, shareReading blank=True 등등 추가

null4848 committed:
feat: 리뷰 수정

수정 버튼 클릭시 -> 정보 전달
리뷰 작성 모달, 리뷰 수정 모달 따로 관리
본인이 작성한 리뷰만 수정할수 잇게,
DB에서 사진 가져오고,
사진도 수정가능하게 ..

null4848 committed:
feat: 리뷰 답글 수정 부분 html 구현

kb4954 committed:
feat: 내그룹리스트 그룹삭제가능

추후 페이지네이터 수정

hwk3005 committed:
250710 백엔드_member브랜치 머지, 로그인 적용完, 에러수정完

0. member into shareMain 브랜치 머지 완료

1. 그룹만들기
  0) 로그인 했을 때만 그룹만들기 가능하도록 적용 중
  1) 책 선택-모달창
    - 선택 버튼 클릭 시 모달창 닫히고 책 추가 css, tooltip 수정
    - 미선택하고 완료 버튼 클릭시 입력내용 전체 초기화 에러 (alert, focus)
    - form제출시 선택한 책 이미지가 안 뜨고 다른 책 뜰 때 있음
	(원인: 중복isbn => 해결: title, author정보 추가로 비교함)
  2) 관련 해시태그
    - input 직접입력칸 Enter,버튼 적용 안 됨 (enter클릭시 form완료버튼 클릭한 거로 적용됨) 에러
  3) 그룹 비밀번호
    - 미입력하고 완료 버튼 클릭 시 입력내용 전체 초기화 에러 (alert, focus)

2. 메인페이지
  1) 참여중인 그룹
    - 로그인 했을 때만 표시되도록 기능 적용 完
    - 참여중인 그룹 목록 개수 표시
    - 최대 8개 그룹개수 제한 걸어놓기
    - 로그인 했을 때만 표시
  2) 그룹 목록
    - 검색창: 검색 시 검색내용 포함한 그룹 검색 (id 최신순 정렬)
    - 검색창: css 수정 (공지사항p 통일)
    - 책제목, 그룹장, 소개글 표시 html 수정
    - 그룹 제목, 소개글 부분 너비 고정값 주기

null4848 committed:
feat: 수정 구현

수정, 삭제 버튼 내 리뷰, 답글에만 뜨게
리뷰 수정 후 DB 저장, 표시까지 함

kb4954 committed:
feat:페이지네이터 수정 및 북서치 앱 수정파일 복붙

추후 교환독서 연결, reply db머지후 다시 연결확인

kb4954 committed:
Merge branch '4-feat-book-search-backend' into 5-feat-mypage01_backend

null4848 committed:
feat: 키워드 차트, 리뷰 좋아요 수정

키워드랑 키워드 개수 차트로 표시했고
리뷰 좋아요 새로고침하면 선택햇던 좋아요가 사라지는 이슈 수정함.
-> reveiwlike 모델을 만들어서 따로 저장함.

자꾸 DB가 막혀서
댓글, 리뷰 AJAX로 전송하게끔 주말에 수정해야할듯함

null4848 committed:
Squashed commit of the following:

commit f63572827f48d9dd7d75f02ecb9454e7cf259f1b
Author: hwk3005 <hwk3005@naver.com>
Date:   Thu Jul 10 22:23:18 2025 +0900

    250710 백엔드_member브랜치 머지, 로그인 적용完, 에러수정完

    0. member into shareMain 브랜치 머지 완료

    1. 그룹만들기
      0) 로그인 했을 때만 그룹만들기 가능하도록 적용 중
      1) 책 선택-모달창
        - 선택 버튼 클릭 시 모달창 닫히고 책 추가 css, tooltip 수정
        - 미선택하고 완료 버튼 클릭시 입력내용 전체 초기화 에러 (alert, focus)
        - form제출시 선택한 책 이미지가 안 뜨고 다른 책 뜰 때 있음
    	(원인: 중복isbn => 해결: title, author정보 추가로 비교함)
      2) 관련 해시태그
        - input 직접입력칸 Enter,버튼 적용 안 됨 (enter클릭시 form완료버튼 클릭한 거로 적용됨) 에러
      3) 그룹 비밀번호
        - 미입력하고 완료 버튼 클릭 시 입력내용 전체 초기화 에러 (alert, focus)

    2. 메인페이지
      1) 참여중인 그룹
        - 로그인 했을 때만 표시되도록 기능 적용 完
        - 참여중인 그룹 목록 개수 표시
        - 최대 8개 그룹개수 제한 걸어놓기
        - 로그인 했을 때만 표시
      2) 그룹 목록
        - 검색창: 검색 시 검색내용 포함한 그룹 검색 (id 최신순 정렬)
        - 검색창: css 수정 (공지사항p 통일)
        - 책제목, 그룹장, 소개글 표시 html 수정
        - 그룹 제목, 소개글 부분 너비 고정값 주기

commit cb713c89a8585a7bc133c664310e5eeb8e35fb9b
Author: hwk3005 <hwk3005@naver.com>
Date:   Thu Jul 10 17:26:40 2025 +0900

    250709 백엔드_member브랜치 머지 완료, 로그인 관련 기능 적용 중

    0. member into shareMain 브랜치 머지 완료

    2. 메인페이지
      1) 참여중인 그룹
        - 로그인 했을 때만 표시되도록 기능 적용 완료

commit 9aba59670b0fdb967a88995228e900de2dbc91b4
Merge: b5ced5b 5fcffbe
Author: hwk3005 <hwk3005@naver.com>
Date:   Thu Jul 10 16:39:35 2025 +0900

    Merge remote-tracking branch 'origin/8-feat-member' into 7-feat-share_main-addgroup-backend

commit b5ced5b703e51cca58ddb8ca8f07fc5e3227e02d
Author: hwk3005 <hwk3005@naver.com>
Date:   Thu Jul 10 16:17:43 2025 +0900

    member 머지 충돌 해결

commit db64a9c81ee234d7ac31c940bed356e86283966c
Author: hwk3005 <hwk3005@naver.com>
Date:   Thu Jul 10 16:16:46 2025 +0900

    Fix merge conflict in Share_AddGroup.html

commit 6369949349060fcf20c7f15a423c82542fa705e6
Author: hwk3005 <hwk3005@naver.com>
Date:   Thu Jul 10 15:59:33 2025 +0900

    250709 백엔드_Main 검색창 기능 구현, 로그인 관련 기능 적용 중

    1. 그룹만들기
      1) 책 선택-모달창
        - 선택 버튼 클릭 시 모달창 닫히고 책 추가 css, tooltip 수정
      2) 로그인 했을 때만 그룹만들기 가능하도록 적용 중

    2. 메인페이지
      1) 참여중인 그룹
        - 로그인 했을 때만 표시되도록 기능 적용 중 (member)
      2) 그룹 목록
        - 검색창: 검색 시 검색내용 포함한 그룹 검색 (id 최신순 정렬)
        - 검색창: css 수정 (공지사항p 통일)

commit a9867b10f620a1aa82f1c4d727fae244b30bb2e9
Author: hwk3005 <hwk3005@naver.com>
Date:   Wed Jul 9 20:55:04 2025 +0900

    250709 교환독서 title 적용

    250709 교환독서 title 적용

commit 0052ff20260c4eb39a7d37ab23ef31745042e231
Author: hwk3005 <hwk3005@naver.com>
Date:   Wed Jul 9 20:50:06 2025 +0900

    250709 백엔드_그룹만들기 백엔드 구현 완료(도서api연결, Form제출시 DB최종저장)

    1. 그룹만들기
      1) 책 선택-모달창
        - 도서 api 연결
        - 책제목/저자 등 키워드 검색 시 검색목록 표시
        - 책 선택버튼 클릭 시 모달창 닫히고 책 선택됨 (변경도 가능)
      2) Form제출 시 shareMain-ReadingGroup db에 최종 저장

    2. 메인페이지
      1) 참여중인 그룹
        - 만든 그룹 추가
        - 책 이미지 안 띄워지던 오류 해결
      2) 그룹 목록
        - 만든 그룹 최신순으로 정렬
        - 해시태그, 책 이미지 안 띄워지던 오류 해결

commit 5fcffbe0eba8531b8cf073b80d2ef4231da9f7b0
Author: resader1 <resader40@gmail.com>
Date:   Wed Jul 9 15:47:35 2025 +0900

    0709pm

    중복확인재진행 경고창 제거
    성별 강제선택 제거

commit 379049b1f70996971b8dae6648784a57bee47c93
Author: hwk3005 <hwk3005@naver.com>
Date:   Tue Jul 8 21:56:58 2025 +0900

    250708 백엔드_Form제출시 DB저장, 그룹목록에 추가한 그룹 표시

    1. 그룹만들기 Form제출시
      1) DB에 저장 완료
      2) 그룹목록, 참여 중인 그룹에 표시 완료

    2.  그룹만들기-완료 버튼 클릭 시 필수항목 미입력 시 alert, focus 적용

    3. 그룹목록-해시태그 css 수정

commit 91c8bb498236abb0b1c8d6499ebd9dfbf91231cf
Author: resader1 <resader40@gmail.com>
Date:   Tue Jul 8 17:14:21 2025 +0900

    이용약관 기능추가

    join1 모두 동의 기능 추가

commit e89d3f1ab0a6eddb75b15a17610dafbb6ddd748f
Author: resader1 <resader40@gmail.com>
Date:   Tue Jul 8 16:58:27 2025 +0900

    회원가입 취소시 홈으로

    join2.js 223줄 수정

commit b301226d2e4745b6253d5a21e1d50c4da27a32a5
Author: hwk3005 <hwk3005@naver.com>
Date:   Tue Jul 8 16:33:53 2025 +0900

    250708 백엔드_html 수정본 업로드

    250708 백엔드_html 수정본 업로드

commit c860e8070e80128282293cf22b4117b55652deab
Author: resader1 <resader40@gmail.com>
Date:   Tue Jul 8 12:18:09 2025 +0900

    회원정보페이지 재추가

    member/update/user_id 입력시 진입 가능
    취소 버튼 클릭 - confirm(수정 취소?) 수락시 뒤로가기

    미로그인 상태에서 진입시 - 로그인 하라는 경고창 뜨고 login페이지로 넘어감

    타 계정 정보창으로 진입시 - 본인 정보만 수정 가능하다는 경고창 뜨고 홈으로 넘어감(기존엔 로그인 페이지로 넘어갔으나 로그인 된 상태에서 로그인 창으로 넘어가지는 현상으로 인하여 수정)

commit 70d8626994f584812450fd4251a3800589611018
Author: resader1 <resader40@gmail.com>
Date:   Tue Jul 8 10:33:30 2025 +0900

    0708am

    전 페이지 base.html extends로 변경.
    js추출.

    회원가입 id중복확인 기능 추가
    ---
    1.중복확인 미진행시 회원가입 진행 불가 및 경고창 팝업
    2.중복확인 후 아이디 변경시 중복확인 재진행 안내문 팝업 후 중복확인 기록 삭제

commit ac1ff69556f8a59e267efb48c9f576c1e6f4c53d
Author: resader1 <resader40@gmail.com>
Date:   Mon Jul 7 14:21:01 2025 +0900

    회원정보 페이지 수정

    user_id님 정보수정
    -->
    member.name님 정보 수정

    변경

commit 53c12e147ab153314bf60aff3e1af5e4123d83b6
Author: resader1 <resader40@gmail.com>
Date:   Mon Jul 7 14:13:32 2025 +0900

    오작동 수정

    로그인창 회원가입 버튼 클릭시 로그인 로직 수행되는 현상 수정

commit 3de5768074cf6c9a864e3136d78e2788b8778caa
Author: resader1 <resader40@gmail.com>
Date:   Mon Jul 7 13:04:27 2025 +0900

    1차 끝

    css 최신화.
    아이디/비밀번호 찾기 인증 조건에 이름 추가.
    인라인 스타일 전부 스타일 시트로 정의.
    로그인/회원정보입력/회원정보수정 페이지에 pw input창 자동완성 기능 off.

commit 4fc61313c8c9f888f97b17a818a035c244b56779
Author: hwk3005 <hwk3005@naver.com>
Date:   Fri Jul 4 13:13:25 2025 +0900

    Remove cached .pyc files and ignore pycache

commit 779f45be99c48bdd2946788f15db72ec81b5545a
Author: hwk3005 <hwk3005@naver.com>
Date:   Fri Jul 4 13:12:43 2025 +0900

    Remove pycache files from git tracking

commit 48c5d08aa0760f68b54636915bc4238d82a40ff4
Author: hwk3005 <hwk3005@naver.com>
Date:   Thu Jul 3 18:42:58 2025 +0900

    250703 백엔드_기본 app, urls 등 기본 설정, a태그

    1. app생성: Share_Main, Share_AddGroup
    2. app_webGBGB: settings, urls 설정
    3. app_base: urls, views 설정

    4. app_shareMain
      1) urls, views, templates 설정
      2) Share_Main.html
        (1) a태그 추가 - 그룹만들기 버튼 클릭 시 page이동
        (2) base, css, static 연결
      3) Share_AddGroup.html
        (1) 완료 버튼 클릭 시 page이동
        (2) base, css, static 연결

    4. app_static: css파일 옮겨놓음

resader1 committed:
Update db.sqlite3

resader1 committed:
회원정보수정 관심부분

돌림

resader1 committed:
Update db.sqlite3

null4848 committed:
setting

null4848 committed:
feat: reading group 가져오기

kb4954 committed:
feat:승민님 브랜치 새로 병합, 펼치기 버튼 수정

리뷰 좋아요 db 가져옴
정보수정 태그부분 변경파일로 변경

resader1 committed:
1

hwk3005 committed:
250711 백엔드_paginator기능, 기타 기능,css,에러 수정

1. 그룹만들기
  1) 책 선택
    - 모달창: paginator 기능 적용 (최대 100권까지만 출력되도록 설정)
    - 모달창: 책 설명 부분 api 끌고 오기 (contents)
  2) 공개여부
    - 비공개 클릭 시 메인페이지 그룹명 첫글자 자리에 비공개 icon 추가

2. 메인페이지
  1) 참여중인 그룹
    - "회원"님 부분에 닉네임 연결
    - 숫자 반복된 group_name - enter 적용 안 됨
    - group_name 최대 2줄까지 자르고 숨기기
    -
  2) 그룹 목록
    - 검색창 - 검색 시 검색내용 포함한 그룹 검색 (그룹장 추가)
    - paginator 기능 적용
    - 해시태그 list 쉼표 붙는 부분 수정 필요
    - group_name 1줄까지, description 2줄까지 자르고 숨기기
    - Title(검색바,만들기 버튼) 너비 css 수정
    - 검색결과 없을시 표시되는 내용 css 수정

sharehyun committed:
메인배너 DB에서 받아와서 출력

메인배너 내용 db에서 출력, 배경 이미지에서 흐리게 설정
공지사항 검색시 "검색결과"출력
base 로그인 후 닉네임 클릭시 오버레이 창 뜸, 고객센터 탭 추가

sharehyun committed:
Merge branch '7-feat-share_main-addgroup-backend' into 3-feat-cscenter-backend

kb4954 committed:
feat: 계정삭제 기능 추가

게정삭제기능
member/member_update.html 수정
member_delete.css
member_delete.js
추가
mypage views.py 수정

sharehyun committed:
111111

null4848 committed:
feat:

조회수 증가 및 쿠키 중복 방지
생성된날짜로부터 5개

hwk3005 committed:
250711 css 추가 수정

2. 메인페이지
  1) 참여중인 그룹
    - 참여중인그룹 미표시(로그아웃)일 때 feed__tabs 상하margin css조정

null4848 committed:
feat: 수정

교환독서에서 더보기 버튼 추가 후 클릭시 교환독서 메인에서 책이름 검색한 페이지로 연결

로그인 안했을 시search에서 검색하면 검색결과 안나오는 거 수정
member는 NONE으로 받고
멤버 없을 시 북마크 버튼 안뜨게
detail에서도 북마크, 리뷰 버튼 안뜨고 미로그인시 리뷰 답글 안보이게 함

api 값 파싱할때 () 이상하게 보이는 거 수정

null4848 committed:
feat

null4848 committed:
feat: 리뷰 수정 이미지

리뷰 수정 이미지
DB 저장
원래 잇던 이미지 삭제
새로운 url 폼으로 받아서 저장

리뷰 버튼 로그인앙ㄴ하면 안뜨게 만들엇는데
alert 뜨도록 이따 수정할예정

null4848 committed:
Merge remote-tracking branch 'origin/5-feat-mypage01_backend' into 4-feat-book-search-backend

null4848 committed:
feat: 마이페이지 병합

null4848 committed:
Update db.sqlite3

resader1 committed:
Create .gitignore

hwk3005 committed:
Remove __pycache__ from Git and update .gitignore

hwk3005 committed:
test 캐시 제거

kb4954 committed:
feat: test

resader1 committed:
Update find1.html

resader1 committed:
Update find1.html

kb4954 committed:
Remove cached pycache/.pyc files

kb4954 committed:
Merge branch '8-feat-member' into 5-feat-mypage01_backend

kb4954 committed:
Merge remote-tracking branch 'origin/3-feat-cscenter-backend' into 5-feat-mypage01_backend

kb4954 committed:
feat:수정

kb4954 committed:
Merge remote-tracking branch 'origin/7-feat-share_main-addgroup-backend' into 5-feat-mypage01_backend

null4848 committed:
feat: 북마크, 썸네일 클릭시 펼치기 접기

kb4954 committed:
feat: 수정

kb4954 committed:
feat: 민철,유현,혜원님 최종본 병합

db 초기화후 진행함

kb4954 committed:
feat:db

null4848 committed:
feat: 리뷰 수정 이미지 부분 수정완료...

kb4954 committed:
feat: js 내리뷰 수정

null4848 committed:
feat: session name 업데이트 코드 추가

null4848 committed:
Merge branch '5-feat-mypage01_backend' into 4-feat-book-search-backend

null4848 committed:
feat: 북마크 부분

null4848 committed:
feat: 수정

로그인 안했을 시 리뷰 버튼 클릭하면 alert 창뜨게 수정
리뷰 작성시 태그, 별점, 리뷰 10자 이상 안썻으면 alert창 뜨게 수정
리뷰에서 이미지 클릭하면 이미지 url로 넘어가게끔 수정함

null4848 committed:
feat: 리뷰 수정, 등록 alert 뜨게 수정

hwk3005 committed:
250714 AddGroup.js 해시태그 if문 수정(한글,영어,숫자 가능)

1. 그룹만들기
  1) 해시태그 직접입력 if문 수정
    - 한글만 가능 → 한글, 영어, 숫자만 입력 가능 (공백,특수문자 불가)

sharehyun committed:
home 수정(준비중 출력)

sharehyun committed:
배너 bullet,nav 색상변경

fafafa

hwk3005 committed:
Merge branch '7-feat-share_main-addgroup-backend' into 4-feat-book-search-backend

sharehyun committed:
Merge branch '4-feat-book-search-backend' into 3-feat-cscenter-backend

hwk3005 commented:
<img width="428" height="543" alt="image" src="https://github.com/user-attachments/assets/f316f46d-09d1-4cdb-a454-ff25961cdc38" />

마이페이지 - 북마크 부분 검색페이지와 비슷하게 책제목,저자 사이 간격, margin 수정하면 좋을 것 같아요!

hwk3005 committed:
250714 tag 모델, Main html, css 조금 수정

1. 그룹만들기
  1) 해시태그
    - tag models - max_length 제거(CharField → TextField 변경)

2. 메인페이지
  1) 참여중인 그룹 - x
  2) 그룹 목록
    - 책제목,그룹장,소개글 html 수정
    - 해시태그 css - cursor: pointer; 주석 처리

sharehyun committed:
5개 추ㅡㄹ력

null4848 committed:
feat: 수정

페이지네이터 정리
삭제할때 confirm
그룹 표시 부분 width 정리
조회수 쿠키 없이 함

null4848 committed:
feat: 홈 업데이트

LatteMeng committed:
FEEDPAGE

1차 완성본(기능구현용도).. 나머지는 열작업중입니다 ㅜㅜ

LatteMeng committed:
교환독서페이지

1차 발표본

null4848 committed:
feat

hwk3005 committed:
250714 max_member form저장 적용完

1. 그룹만들기
  2) 최대 인원수
    - max_member 최대 인원수 hidden input으로 form 저장 적용 完

null4848 committed:
Merge branch '4-feat-book-search-backend' into 14-feat-feedpage

sharehyun committed:
Merge branch '4-feat-book-search-backend' into 3-feat-cscenter-backend

sharehyun committed:
db저장

LatteMeng committed:
Merge remote-tracking branch 'origin/3-feat-cscenter-backend' into 14-feat-feedpage

null4848 committed:
feat: 교환독서 피드, 상세 app 연결

LatteMeng committed:
Merge remote-tracking branch 'origin/4-feat-book-search-backend' into 14-feat-feedpage

resader1 committed:
Update db.sqlite3

hwk3005 committed:
250714 Share_Main - book img클릭시 책 상세페이지 이동 추가

2. 메인페이지
  2) 그룹 목록
    - Share_Main - 그룹목록 - book img클릭 시 책 상세페이지로 이동 추가

null4848 committed:
feat: 이미지 로딩 안될때 연결이미지 변경, shareMain에서 이미지 클릭시 detail 로 이동

LatteMeng committed:
feedpage,feedintro

피드디테일 출력, 페이지연결(디테일페이지)

hwk3005 committed:
250714 tag - hidden input 중복 코드 제거

1. 그룹만들기
  1) 해시태그
    - hidden input 중복 코드 제거

null4848 committed:
Merge branch '7-feat-share_main-addgroup-backend' into 4-feat-book-search-backend

null4848 committed:
feat: 브랜치 병합

null4848 committed:
feat: 북디테일 교환독서 연결

null4848 committed:
feat: 배너 부분 색 변경

null4848 committed:
feat: 리뷰 개수 뜨게함

resader1 committed:
0715 추가

join2
이메일 인증 추가(인증 로직에 이메일 중복 체크 기능 내장)
성별 미선택 radio박스 추가

login, join1 카카오 버튼 추가.
카카오 로그인/회원가입 가능.

null4848 committed:
Update db.sqlite3

resader1 committed:
이미지 추가

null4848 committed:
Merge branch '8-feat-member' into 4-feat-book-search-backend

resader1 committed:
update 날아간거 추가

추가

null4848 committed:
setting

LatteMeng committed:
피드페이지

null4848 committed:
feat:

null4848 committed:
Merge branch '14-feat-feedpage' into 4-feat-book-search-backend

resader1 committed:
3

resader1 committed:
3

null4848 committed:
feat: chatroom js 추가

null4848 committed:
Merge branch '4-feat-book-search-backend' of https://github.com/SMU-GGOBOOK/backend into 4-feat-book-search-backend

null4848 committed:
feat: 세팅, css 북디테일

hwk3005 committed:
250715 chatroom, sns_feed 링크 연결, 태그 에러 해결完

1. 그룹만들기
  1) 해시태그
    - hidden input 잘못 주석처리 해놨던 것 복구 및 에러 해결

2. 메인페이지
  1) 참여중인 그룹
    - 참여중인 그룹 이미지 클릭 시 sns_feed 피드페이지로 이동
  2) 그룹 목록
    - 로그인 안 했을 경우 그룹 클릭 시 alert 띄우며 로그인 페이지로 이동
    - 로그인한 경우 그룹 클릭 시 chatroom페이지로 이동

null4848 committed:
feat: 독서그룹데이터수정

sharehyun committed:
채팅방 디테일 링크연결.defaultimg 수정

null4848 committed:
Merge branch '7-feat-share_main-addgroup-backend' into 4-feat-book-search-backend

sharehyun committed:
Merge branch '4-feat-book-search-backend' into 3-feat-cscenter-backend

hwk3005 committed:
250715 그룹목록 상단에 그룹 총개수 표시

  2) 그룹 목록
    - 그룹목록 상단에 그룹 총개수 표시

hwk3005 committed:
250715 sns_feed 피드페이지로 이동 링크 수정

2. 메인페이지
  1) 참여중인 그룹
    - 참여중인 그룹 이미지 클릭 시 sns_feed 피드페이지로 이동 링크 수정

null4848 committed:
Merge branch '7-feat-share_main-addgroup-backend' into 4-feat-book-search-backend

resader1 committed:
login함수 추가

resader1 committed:
카카오 추가

null4848 committed:
feat: 푸터 수정, 교환독서 url 수정

로그인 시 name 저장

hwk3005 committed:
250715 chatroom, sns_feed 이동 url 수정

resader1 committed:
3

null4848 committed:
feat: 메인페이지 교환독서그룹 인기 도서 순위 출력

null4848 committed:
setting: DB 저장

null4848 committed:
Merge branch '7-feat-share_main-addgroup-backend' into 4-feat-book-search-backend

null4848 committed:
feat: css 수정

resader1 committed:
Update db.sqlite3

hwk3005 committed:
250716 폰트어썸 script 변경

기존 슬비님 폰트어썸 사용량 초과돼서 작동 안 됨
-> 내꺼로 변경

hwk3005 committed:
Merge branch '4-feat-book-search-backend' into 7-feat-share_main-addgroup-backend

kb4954 committed:
feat: 교환독서 피드 연결 ajax

hwk3005 committed:
Merge remote-tracking branch 'origin/8-feat-member' into 7-feat-share_main-addgroup-backend

kb4954 committed:
feat: css 수정

교환독서 타이틀 부분 justify 주석처리

null4848 committed:
feat: 수정

폰트어썸 링크 수정
리뷰 엔터 안먹는 거 수정
비공개 그룹 -> 이모티콘 추가
(메인, 피드 상세 페이지)
메인 배너 클릭시 디테일로 이동하게 - 임시로 해둠
피드 페이지 수정

kb4954 committed:
feat: 리뷰 삭제 버튼 사이즈 수정

차단한 사용자 주석처리함

hwk3005 committed:
250716 검증테스트 중

null4848 committed:
feat: 이미지 모달 수정

chat_feed 부분이랑 bookdetail 이미지클릭시 모달뜨게함 (chat_feed는 원래 됐었는데 html 수정하면서안됨)
chatfeed 이미지 크기 수정

hwk3005 committed:
Merge branch '4-feat-book-search-backend' into 7-feat-share_main-addgroup-backend

null4848 committed:
Update db.sqlite3

null4848 committed:
Merge branch '8-feat-member' into 4-feat-book-search-backend

null4848 committed:
feat: isbn 정보없음 처리

kb4954 committed:
feat: 북마크, 내그룹

이미지 없는 표지일때 나타낼 표지이미지 수정

kb4954 committed:
feat: 북마크 따옴표 추가

hwk3005 committed:
250715 그룹만들기_책선택 form저장 에러 해결

1. 그룹만들기
  1) 책 선택
    - 책 선택시 저자가 공란일때 그룹가입 폼 미전송 에러
    - 책 선택시 책 표지가 없을 때 그룹가입 폼 미전송 에러

null4848 committed:
feat: DB 수정, sns_feed 글 작성시 DB 저장 수정

null4848 committed:
Merge branch '3-feat-cscenter-backend' into 4-feat-book-search-backend

kb4954 committed:
feat: 최종 커밋본

kb4954 committed:
feat: 데이터

resader1 committed:
update 수정

해시태그 정상화

resader1 committed:
Update db.sqlite3

sharehyun committed:
neews 추가

sharehyun committed:
Merge branch '4-feat-book-search-backend' into 3-feat-cscenter-backend

resader1 committed:
Update member_update.js

resader1 committed:
로그인 유지 안되는 현상

카카오 로그인시 localhost로 리다이렉트 되어, 페이지 재접속시 127.0.0.1로 처리되는 과정에서 로그인 유지 되지 않는 현상 처리

resader1 committed:
Update models.py

LatteMeng committed:
그룹방 상세보기

기능 추가(비공개방 비번 입력 후 입장)

sharehyun committed:
꼬북늬우스

sharehyun committed:
메인페이지 교환독서방 수정

null4848 committed:
feat: 교환독서 그룹 상세, 피드 수정

db 불러와서 출력으로 수정
기존 API 코드 삭제
가입 요청 코드 추가

null4848 committed:
feat: 메세지 alert으로 뜨게 수정

sharehyun committed:
헤더 꼬북뉴스

kb4954 committed:
Merge branch '4-feat-book-search-backend' into 5-feat-mypage01_backend

null4848 committed:
feat: setting 수정

null4848 committed:
Merge branch '3-feat-cscenter-backend' of https://github.com/SMU-GGOBOOK/backend into 3-feat-cscenter-backend

null4848 committed:
Merge branch '3-feat-cscenter-backend' into 4-feat-book-search-backend

null4848 committed:
Merge branch '8-feat-member' into 4-feat-book-search-backend

null4848 committed:
Merge branch '5-feat-mypage01_backend' into 4-feat-book-search-backend

null4848 committed:
Merge branch '7-feat-share_main-addgroup-backend' into 4-feat-book-search-backend

null4848 committed:
setting

sharehyun committed:
css수정

hwk3005 committed:
250717 footer 리뷰개수 적용 (그룹만들기 화면)

250717 footer 리뷰개수 적용 (그룹만들기 화면)

sharehyun committed:
Merge branch '3-feat-cscenter-backend' of https://github.com/SMU-GGOBOOK/backend into 3-feat-cscenter-backend

sharehyun committed:
음냐

null4848 committed:
Merge branch '3-feat-cscenter-backend' into 4-feat-book-search-backend

resader1 committed:
3

resader1 committed:
3

LatteMeng committed:
그룹방 상세페이지 수정

그룹에 가입한 멤버 수 sharemain_reqding-group-member에서 group_id로 member_id 총합 불러오기

hwk3005 committed:
250717 book.cover db, form저장 오류 해결

1. 그룹만들기
  2) book.cover db, form저장 오류 해결

sharehyun committed:
통계페이지, cscenter

공지사항 리스트 검색시 검색어도 띄움
1:1문의 로그인 세션 없이 클릭시 이전페이지가 아니라 로그인페이지로 이동
통계페이지 css작업(분리예정)

sharehyun committed:
ㅋㅋ오타

resader1 committed:
f

카카오 회원가입시 닉네임 변경 가능하게 수정.
회원정보 수정에서 email값 변경 불가능하게 수정.(이유 :로그인 인증이 추가됨)

sharehyun committed:
db 추가

kb4954 committed:
feat: db

null4848 committed:
setting

sharehyun committed:
일부수정

sharehyun committed:
Merge branch '8-feat-member' into 3-feat-cscenter-backend

sharehyun committed:
Merge branch '7-feat-share_main-addgroup-backend' into 3-feat-cscenter-backend

kb4954 committed:
Merge branch '4-feat-book-search-backend' into 5-feat-mypage01_backend

kb4954 committed:
feat: 승민님꺼 머지

sharehyun committed:
Merge branch '5-feat-mypage01_backend' into 3-feat-cscenter-backend

kb4954 committed:
Merge branch '3-feat-cscenter-backend' into 5-feat-mypage01_backend

sharehyun committed:
member,mypage merge함

sharehyun committed:
121212121

kb4954 committed:
Merge branch '3-feat-cscenter-backend' into 5-feat-mypage01_backend

kb4954 committed:
Merge branch '3-feat-cscenter-backend' into 5-feat-mypage01_backend

kb4954 committed:
feat: 유현님 파일 받아옴

resader1 committed:
32

kb4954 committed:
feat

kb4954 committed:
Merge branch '14-feat-feedpage' into 5-feat-mypage01_backend

kb4954 committed:
feat: 수정

sharehyun committed:
이미지 없을때 디폴트이미지안뜨는거

null4848 committed:
feat: 피드페이지, 상세페이지 수정

post db에 저장

kb4954 committed:
Merge branch '4-feat-book-search-backend' into 5-feat-mypage01_backend

sharehyun committed:
Merge branch '4-feat-book-search-backend' into 3-feat-cscenter-backend

kb4954 committed:
feat: 민성님 브랜치 병합

sharehyun committed:
수정쓰

null4848 committed:
feat: 댓글 DB, post DB 연결 완료

좋아요 DB는 조금 더 수정이 필요할 듯함

null4848 committed:
Merge branch '3-feat-cscenter-backend' into 4-feat-book-search-backend

null4848 committed:
Merge branch '5-feat-mypage01_backend' into 4-feat-book-search-backend

null4848 committed:
Merge branch '8-feat-member' into 4-feat-book-search-backend

kb4954 committed:
feat: 백엔드 확인

kb4954 committed:
Merge remote-tracking branch 'origin/4-feat-book-search-backend' into 5-feat-mypage01_backend

kb4954 committed:
feat: 메인페이지
